### PR TITLE
Disable service replicas with development mode flag

### LINKED ISSUE #258: Introduce 'development mode' consistently across all charts
**Is this a bug report or feature request?** (choose one):  Feature request

**Kubernetes Version** (output of `kubectl version`):  Any

**Helm Client and Tiller Versions** (output of `helm version`): Any

**Development or Deployment Environment?**:  Development

**Release Tag or Master**:

**Expected Behavior**:  Ability to consistently set flags across all charts for developer mode to control the number of replicas where necessary and to circumvent the need for PVCs

**What Actually Happened**:

**How to Reproduce the Issue** (as minimally as possible):

**Any Additional Comments**:  

While some charts have the ability to set a development mode flag, others do not.  In order to maintain a positive developer experience for OpenStack-Helm going forward, we should consistently apply the ability to set a development mode flag across all future charts.

### PR DESCRIPTION
**Pull request by wilkers-steve**
Disable service replicas with development mode flag in all charts

To remain consistent with other services, this adds a development
mode flag to reduce the number of replicas to 1.

**What is the purpose of this pull request?**:  Disables replicas for a service when development flag is set to `enabled`

**What issue does this pull request address?**: Partially addresses #258

**Notes for reviewers to consider**:

**Specific reviewers for pull request**:

### CODE REVIEW

File: nova/values.yaml [hunk 1]
```diff
@@ -17,14 +17,18 @@
...
   control_node_selector_key: openstack-control-plane
   control_node_selector_value: enabled
   compute_node_selector_key: openstack-compute-node
   compute_node_selector_value: enabled
 
-control_replicas: 1
-compute_replicas: 1
+replicas:
+  control: 1
+  compute: 1
```
Comment (intlabs) [inline nova/values.yaml]: Is `compute` used? If not can we remove it?

**Comment by wilkers-steve** [comment]
@v1k0d3n @intlabs related to our discussion re: rethinking development mode, do we want to close this PR out?  After some thought, I'm not a fan of the idea of controlling replicas with a development flag.

**Comment by v1k0d3n** [comment]
yup, i'll let you close it out. i think that if we just stick with a developer focused config-override on the client side, rather than with debt in the form of conditionals we'll be in a lot better shape (imo, and as we've discussed collectively).

**Comment by wilkers-steve** [comment]
Absolutely agreed. :+1:

**Pull request closed without merging**